torch/lib/c10d/logger.cpp

```diff
@@ -204,6 +204,12 @@ void Logger::set_runtime_stats_and_log() {
     ddp_logging_data_->rebuilt_bucket_sizes = get_bucket_sizes();
   }
 
+  // reset perforamnce stats for current iteration.
```

Comment:
Nit: It will be better to wrap up these lines a as `reset_perf_stats` function, and you can put anything that needs to be reset per iteration in it in the future.